### PR TITLE
set num data for scale rather then shown all datas

### PR DESCRIPTION
**Pull request by guochaorong**

### CODE REVIEW

File: web/view.py [hunk 1]
```diff
@@ -349,7 +349,7 @@ def logic(self):
         # should be sorted by freshness
         commits = get_commits()
         kpis = {}
-        for commit in commits:
+        for commit in commits[:-self.N:-1]:
```
Comment (Superjomn) [inline web/view.py:352]: 需要确认下这里是按时间从新往旧排序过的，取 top N 才有效

Comment (guochaorong) [reply web/view.py:352]: 恩恩， 确认过~

Comment (Superjomn) [reply web/view.py:352]: 加个按时间排序？最好验证下

Comment (guochaorong) [reply web/view.py:352]: ok. I have added the sorting by date in this pr:
#59

**Pull request closed without merging**